conformance/conformance.proto

```diff
@@ -57,11 +57,13 @@ option java_package = "com.google.protobuf.conformance";
 //   2. parse the protobuf or JSON payload in "payload" (which may fail)
 //   3. if the parse succeeded, serialize the message in the requested format.
 message ConformanceRequest {
+  string test_name = 1;
```

Comment:
If the testee crashes, shouldn't the test runner be the one who logs the crash information with the test name? I agree the names is part of the public interface but it's the public interface of the test runner for end users, not the public interface between the test runner and the testee. For example, you only provide the failure list to the test runner, not the testee. I couldn't think of a case where the name must exists in the test protocol and it leads me to believe it should not be there.